infra/base-images/base-builder/install_go.sh

```diff
@@ -28,11 +28,7 @@ go install github.com/mdempsky/go114-fuzz-build@latest
 ln -s $GOPATH/bin/go114-fuzz-build $GOPATH/bin/go-fuzz
 
 cd /tmp
-git clone https://github.com/AdamKorcz/go-118-fuzz-build
+git clone --branch=dev https://github.com/AdamKorcz/go-118-fuzz-build
```

Comment:
Should we expect the dev branch changes to be merged into the main branch soon?

Reply:
Yes. That should happen by end of month.